SUMMARY.md

```diff
@@ -164,6 +165,7 @@
   * [sources.xml](collector/configuration/sources.xml.md)
   * [Tags in LiveRig Collector](collector/configuration/tags-in-liverig-collector.md)
   * [Certificate-based authentication for WITSML HTTPS sources](collector/configuration/certificate-based-authentication-for-witsml-https-sources.md)
+  * [store.json](collector/configuration/store.json.md)
```

Comment:
Missing modbus.json.md under the configuration in SUMMARY.md listing. 

Sorry folks, I forget to explain the SUMMARY.md is important to keep consistent to the new tree of .md, actually when editing by gitbok UI , it ensures the consistency but here at github integration we need to be very careful to those links.

Bold rule:
- if any page is removed, please remove from summary
- if any page is created, please add to the summary

Reply:
Fixed in the commit 18ed7bf0cbbe79b93720c7873bc3a625920ed593